src/Templates.xml

```diff
@@ -394,17 +397,18 @@
       <Package Name="Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore" Version="1.1.0" />
       <Package Name="Microsoft.AspNetCore.Identity.EntityFrameworkCore" Version="1.1.0"/>
       <Package Name="Microsoft.AspNetCore.Mvc" Version="1.1.0" />
+      <Package Name="Microsoft.AspNetCore.Mvc.Razor.ViewCompilation" Version="1.1.0-msbuild3-final" />
       <Package Name="Microsoft.AspNetCore.StaticFiles" Version="1.1.0" />
       <Package Name="Microsoft.EntityFrameworkCore.Design" Version="1.1.0" />
```

Comment:
This could be removed. It's brought in by `Microsoft.EntityFrameworkCore.Tools` now.